nixos/modules/hardware/keyboard/uhk.nix

```
@@ -0,0 +1,16 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.hardware.keyboard.uhk;
+in
+{
+  options.hardware.keyboard.uhk = {
+    enable = mkEnableOption "non-root access to the firmware of UHK keyboards";
```

```suggestion
    enable = mkEnableOption ''
    non-root access to the firmware of UHK keyboards.
    You need it when you want to flash a new firmware on the keyboard.
    Access to the keyboard is granted to users in the "input" group.
    You may want to install the uhk-agent package.
    '';
```
Here seems like a better spot to put it (adapted from `hardware.zsa`)